modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,86 @@
+
+. Enable `ssh` (secure shell) in the ControlPanel. In the navigation bar on the left side, select menu:Networks & File Services[Telnet / SSH]. Check `Allow SSH connection` and specify the port number (default 22). Next, check `Enable SFTP`. Once you click btn:[Apply], your admin user can log in to your NAS remotely.
+
+== Installation of ownCloud
+
+Go to the AppCenter and select QNAP Store.
+
+Search for ownCloud(_10.8.0_x86_64.qpkg) and click btn:[Install]. Confirm that you want to install this package. During installation you'll see a progress bar displaying the percentage.
+
+If you forgot to install the Container Station, an error message will pop up. Click on the link "System Event Log" in the notification window to find out what actually went wrong or hit btn:[OK].
+
+Via the QuLog Center utility you can also check the log entries.
+
+The event notifications in the top tool bar will also tell you what has gone wrong.
+
+Install the Container Station and then start the installation of ownCloud again. After a successful installation, ownCloud 10.8.0 will show up in the AppCenter under menu:QNAP Store[My Apps]. Click on btn:[Open] and proceed with the configuration.
+
+== Configuration
+
+=== First steps
+
+Log in as user `admin`. The default password is `admin` until you change it. In the upper right-hand corner, click on menu:Admin[Settings]. In the left hand navigation bar you find personal configuration options and admin-specific options. Under menu:Personal[General] you can change the password of you admin user.
+
+For more information on the Web interface, refer to the xref:user_manual:webinterface.adoc[Web UI] documentation.
+
+=== Configure your email.
+
+Email is an essential tool in ownCloud for notifying users, user registration, password resets, etc. Logged-in as admin, you can configure email in the menu:Admin[General] settings.
+
+For more information, refer to the section xref:configuration/server/email_configuration.adoc[Email Configuration].
+
+=== Cron
+
+On the same page (menu:Admin[General]) you find the Cron settings.
+
+Make sure Cron remains enabled. This daemon takes care of service jobs in 15-minute intervals.
+
+NOTE: In general, we recommend to keep the default settings in the admin section.
+
+== User Management
+
+== Enterprise Edition and Licensing
+
+If you need more users than five or want to use enterprise features, obtain an Enterprise License and possibly additonal user packages via the QNAP Store and activate them in the QNAP License Manager.
+
+Licenses are valid for a year. If you don't renew them, only the first five users created will remain enabled as well as non-enterprise apps. Should you decide to buy an Enterprise license and user packages again, you can enable the users you want to become active again in the Web interface
+
+== Enabling ownCloud Apps
+
+With the free community edition, you can only use apps with an ocl license. Look out for 'ocl' in the ownCloud Marketplace ...
```

Comment:
Make a link for the marketplace.
Searching for ocl, nothing gets returned which might frustrate users...

Reply:
Yep, still need to check into this.